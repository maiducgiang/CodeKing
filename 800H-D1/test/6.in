10
55
6 6 26 9 34 39 29 10 54 36 8 16 53 15 42 50 41 12 32 18 33 13 20 28 2 26 50 19 51 22 53 29 27 2 51 20 27 13 5 6 35 53 5 55 17 39 30 20 6 46 46 26 16 29 39 
68
4 61 8 35 37 17 29 68 61 65 19 62 36 4 67 37 1 64 7 21 53 40 33 55 17 27 27 15 8 52 63 2 21 33 34 44 3 50 37 31 47 40 17 21 34 12 51 57 21 57 63 66 28 40 4 43 5 5 34 59 9 29 28 18 64 53 11 43 
68
18 40 23 27 17 13 19 10 47 13 11 2 41 30 42 19 23 47 59 24 44 66 5 31 26 59 25 28 57 15 24 47 66 66 20 3 33 58 1 22 33 44 67 46 68 27 34 41 1 38 36 39 40 5 13 58 8 49 2 39 27 53 7 46 63 21 52 3 
49
6 32 10 39 31 40 23 8 12 40 31 9 26 35 31 11 44 9 23 8 47 49 48 35 36 16 33 11 36 22 30 46 28 43 7 5 23 2 34 6 6 26 36 41 33 41 11 25 31 
43
35 28 8 21 3 32 21 21 10 18 10 35 21 9 23 13 6 23 27 28 16 1 10 11 2 10 23 36 11 38 30 20 23 15 31 10 23 30 6 30 31 4 9 
43
13 10 22 30 8 23 6 42 38 36 10 22 27 6 31 34 37 32 23 9 36 20 15 32 27 14 15 35 4 38 13 25 34 38 35 2 27 27 10 4 14 34 1 
50
4 38 45 42 22 12 37 24 24 30 23 46 34 26 33 45 21 34 35 26 18 40 28 27 19 34 17 24 24 17 33 41 27 38 21 22 48 4 2 6 39 31 24 19 21 29 42 40 37 11 
59
23 39 51 18 33 35 57 59 5 55 6 26 27 44 56 14 28 34 15 27 35 35 1 28 58 58 53 58 50 7 25 38 7 25 32 15 59 38 38 14 41 50 17 2 39 56 19 8 26 45 51 6 36 47 15 41 19 6 32 
45
12 30 34 26 37 27 36 11 45 3 45 30 25 35 10 8 2 25 17 39 9 13 32 27 41 39 41 33 21 15 8 14 11 35 10 8 32 38 44 18 37 18 28 20 35 
64
12 40 58 37 8 2 51 64 6 59 60 58 53 53 10 6 26 22 22 36 3 21 42 11 47 48 15 4 31 49 7 44 37 15 59 33 45 51 5 62 53 64 12 21 64 41 53 6 16 28 61 22 33 4 48 18 10 48 64 48 36 5 61 37 
