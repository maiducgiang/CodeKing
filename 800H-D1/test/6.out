13
17
24
23
22
22
29
37
23
29
